extension/src/test/suite/patch.test.ts

```diff
@@ -14,11 +14,11 @@ import { STUDIO_ENDPOINT, registerPatchCommand } from '../../patch'
 import { AvailableCommands } from '../../commands/internal'
 import expShowFixture from '../fixtures/expShow/base/output'
 import { dvcDemoPath } from '../util'
-import { ExperimentFields } from '../../cli/dvc/contract'
 import { Toast } from '../../vscode/toast'
 import { Modal } from '../../vscode/modal'
 import { Response } from '../../vscode/response'
 import { RegisteredCommands } from '../../commands/external'
+import { ExpData } from '../../cli/dvc/contract'
```

Comment:
[F] These tests will be getting deleted as the patch is being replaced by `exp push`.